Sources/Utility/ArgumentParser.swift

```diff
@@ -193,15 +214,15 @@ public final class PositionalArgument<Kind>: ArgumentProtocol {
 /// A type-erased argument.
 ///
 /// Note: Only used for argument parsing purpose.
-fileprivate final class AnyArgument: ArgumentProtocol, CustomStringConvertible {
+public final class AnyArgument: ArgumentProtocol, CustomStringConvertible {
```

Comment:
The script generators should be a part of ArgumentParser so this doesn't need to be public